plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -436,13 +436,6 @@ func ClusterRoles() []rbac.ClusterRole {
 				rbac.NewRule(Read...).Groups(legacyGroup).Resources("persistentvolumeclaims", "persistentvolumes").RuleOrDie(),
 			},
 		},
-		{
-			// a role to use for the kube-dns pod
-			ObjectMeta: metav1.ObjectMeta{Name: "system:kube-dns"},
```

Comment:
Per our deprecation policy, this change will need to wait at least two releases following an announcement https://kubernetes.io/docs/reference/deprecation-policy/

Reply:
@ericchiang thanks, I think I should follow https://github.com/kubernetes/kubernetes/pull/49638 that keep the clusterrolebinding & clusterrole, clean the subject field and mark it as deprecated?

Reply:
https://github.com/kubernetes/kubernetes/pull/49638 was a slightly special case because we wanted to keep the ClusterRole around. I actually don't remember why we only removed the subject field.

I would add the RBAC resources to the `cluster/addons/dns/` folder for this PR. Then follow up after the deprecation grace period with a PR to remove the bootstrap ClusterRole and ClusterRoleBinding.

Reply:
ok, I will keep the clusterrole/clusterolebinding with a deprecate comment.